src/ui_basic/panel.cc

```diff
@@ -1700,7 +1700,7 @@ NamedPanel::NamedPanel(Panel* const nparent,
      name_(name),
      hyperlink_subscriber_(
         Notifications::subscribe<NoteHyperlink>([this](const NoteHyperlink& note) {
-	        if (name_ == note.target) {
+	        if (name_.find(note.target, 0) == 0) {
```

Comment:
```suggestion
	        if (starts_with(name_, note.target)) {
```

Reply:
Thanks, that was the function I searched for :). The content should show up now, I cleaned up one line too much before pushing.